cluster-provision/gocli/opts/rookceph/manifests/cluster-test.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
 spec:
   dataDirHostPath: /var/lib/rook
   cephVersion:
-    image: quay.io/ceph/ceph:v19
+    image: quay.io/ceph/ceph:v19.2.1-20250202
```

Comment:
Also, are you sure it's ceph image that's bringing in the new RBAC requirement? Wouldn't it be some higher level rook component?

Reply:
Well I think we need to pin the rook image too - as we are using the `master` tag there.

Reply:
I don't think pinning ceph makes sense tbh, we should be fine with anything they push to stable. For other components - sure

Reply:
Fair enough - I was trying to see how this got broken and both ceph & rook are using "floating" tags - is there a reason to not pin rook?

Reply:
Ah I see the problem. I accidentally used a floating tag for rook in the last bump.
Should probably do
```diff
       containers:
         - name: rook-ceph-operator
-          image: docker.io/rook/ceph:master
+          image: docker.io/rook/ceph:v1.16.6
```

Reply:
(how is dockerhub not a problem for us??)

Reply:
the images are prepulled into the node images so only pulled at publish. 

but we should try to use quay where we can - I will mirror that image over to our kubevirtci quay repo and create a new PR to pin rook.

My question on this one, could ceph bugs not still be introduced by this floating tag that may introduce flakes/failures that are hard to troubleshoot as we may not think to look at what has changed in ceph and may focus on kubevirt changes?

Reply:
I think the risk is really low for this one tbh. That argument stands for rook though. I doubt rook will ever not work with it's corresponding ceph minor release.

Reply:
Thanks @akalenyu  - will close this one and open a new one for rook when I have the image mirrored.